posts/call-for-maintainers-2020.md

```diff
@@ -0,0 +1,17 @@
+**We're looking for new Track Maintainers. Interested? Read on!**
+
+Exercism is powered by the energy of many volunteers, especially the [track maintainers](https://exercism.io/team/maintainers) who are responsible for creating and managing our Language Tracks.
+
+We're about to start a big push towards version 3 (v3) of Exercism. Getting all the tracks ready for v3 is going to take a big community effort, and we're looking to add a few more maintainers to oversee and manage that task.
+
+In the push to v3, our maintainers will be responsible for:
+- **Designing the track:** Working out what someone needs to learn to become fluent in a language, and mapping this out as a pathway.
+- **Decision making:** Making design decisions around the tooling the track uses, the standards it conforms, the defaults it uses, etc.
+- **Encouraging and supporting contributions:** Providing a welcoming environment for people that want to help out, answering their questions, etc
+- **Reviewing PRs:** A lot of the acutal work of building out v3 tracks will be done by the community. Maintainers act as gatekeepers, reviewing the work that the community does and ensuring the track's standards are enforced.
+
+This might all sound a bit daunting but our more experienced team will help you get started and hold your hand as you learn the ropes. Ideally you need to be able to commit 3hrs per week (preferably spread over a week) for about 6 months, although obviously life ebs and flows, and some weeks might work better than others. You'll also prefably have some experience of Exercism - either as a mentor or having contributed to the codebase. This isn't strictly a requirement, but it's nice to know you've cared about Exercism for a while.
```

Comment:
```suggestion
This might all sound a bit daunting but our more experienced team will help you get started and hold your hand as you learn the ropes. Ideally you need to be able to commit 3hrs per week (preferably spread over a week) for about 6 months, although obviously life ebs and flows, and some weeks might work better than others. You'll also preferrably have some experience of Exercism - either as a mentor or having contributed to the codebase. This isn't strictly a requirement, but it's nice to know you've cared about Exercism for a while.
```

Reply:
> it's nice to know you've cared about Exercism for a while.

Does this add any value over just having:

> This isn't strictly a requirement though.

Reply:
I'm not sure. I think it's nice to explain why it matters but I could go either way.